dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -470,21 +469,11 @@ private DockstoreTool getContainer() {
         tag.setImageId("123456");
         tag.setCwlPath(c.getDefaultCwlPath());
         tag.setWdlPath(c.getDefaultWdlPath());
-        // construct source files
-        SourceFile fileCWL = new SourceFile();
-        fileCWL.setContent("cwlstuff");
-        fileCWL.setType(SourceFile.TypeEnum.DOCKSTORE_CWL);
-        fileCWL.setPath("/dockstore.cwl");
-        fileCWL.setAbsolutePath("/dockstore.cwl");
-        List<SourceFile> list = new ArrayList<>();
-        list.add(fileCWL);
-        tag.setSourceFiles(list);
-        SourceFile fileDockerFile = new SourceFile();
-        fileDockerFile.setContent("dockerstuff");
-        fileDockerFile.setType(SourceFile.TypeEnum.DOCKERFILE);
-        fileDockerFile.setPath("/Dockerfile");
-        fileDockerFile.setAbsolutePath("/Dockerfile");
-        tag.getSourceFiles().add(fileDockerFile);
+        List<String> list = new ArrayList<>();
+        list.add("/Dockstore.cwl");
+        tag.setVerifiedSources(list);
```

Comment:
This was adding to sourceFiles before, now it's adding to verifiedSources. Is this on purpose?

Reply:
fixed by Richard Hansen